pwa-devdocs/scripts/create-reference-docs/config/index.js

```diff
@@ -0,0 +1,10 @@
+const peregrineFiles = require('./peregrine');
+const veniaConceptFiles = require('./venia-concept');
```

Comment:
Are you sure this gets all the files? I was expecting [node-glob](https://github.com/isaacs/node-glob) or similar here.

Reply:
I wanted to be explicit on which files I wanted to process, since we only really care about the publicly available components.
This way unnecessary files aren't processed and generated.